shipping/src/main/java/com/instana/robotshop/shipping/Controller.java

```diff
@@ -95,36 +73,42 @@ public String health() {
 
     @GetMapping("/count")
     public String count() {
-        //Histogram.Timer requestTimer = get_resp_shipping_count.startTimer();
-        //double start = System.currentTimeMillis();
+        // Histogram.Timer requestTimer = requestLatency.startTimer();
+        long start = System.currentTimeMillis();
         long count = cityrepo.count();
-        //double finish = System.currentTimeMillis();
-        //double timeElapsed = finish - start;
+        long finish = System.currentTimeMillis();
+        long timeElapsed = finish - start;
+        meterRegistry.timer("rt_shipping_get_count").record(Duration.ofMillis(timeElapsed));
         //get_resp_shipping_count.observe(timeElapsed);
-        //requestTimer.observeDuration();
+        // requestTimer.observeDuration(); 
         return String.valueOf(count);
     }
 
     @GetMapping("/codes")
     public Iterable<Code> codes() {
+        long start = System.currentTimeMillis();
         logger.info("all codes");
-
         Iterable<Code> codes = coderepo.findAll(Sort.by(Sort.Direction.ASC, "name"));
-
+        long finish = System.currentTimeMillis();
+        long timeElapsed = finish - start;
+        meterRegistry.timer("rt_shipping_get_code").record(Duration.ofMillis(timeElapsed));
         return codes;
     }
 
     @GetMapping("/cities/{code}")
     public List<City> cities(@PathVariable String code) {
+        long start = System.currentTimeMillis();
         logger.info("cities by code {}", code);
-
         List<City> cities = cityrepo.findByCode(code);
-
+        long finish = System.currentTimeMillis();
+        long timeElapsed = finish - start;
+        meterRegistry.timer("rt_shipping_get_citiescode").record(Duration.ofMillis(timeElapsed));
```

Comment:
Change metric name.

If you want to mention the endpoint do so as this https://github.com/drac98/robot-shop/commit/33299e99beb6cf019729adc7f025d8315eaf91d4#diff-89ce88dce7cb771c98606f200db1fde635332a81bdb99bf3d236f7a1034a31a5R33

See the following metric names,
rt_user_get_mongo_checkid
rt_user_get_mongo_users
rt_user_get_mongo_history